transitmatters/t-performance-dash
# feature: Service hours chart

## Motivation

We want to graph the total number of (scheduled and delivered) service hours for Red, Orange, and Blue lines. This is a complementary metric to trip counts, which also happens to be more robust to shuttling.

## Changes

Adds a new graph to `/<line>/service`:

![image](https://github.com/transitmatters/t-performance-dash/assets/2208769/2532ed1e-8a65-4bb7-acbc-d26c52b352b8)

For now weekly/monthly aggregation is done on-the-fly instead of using agg tables, so large queries (2+ years) sometimes fail. There's also no reliable data before about 2018.

Also: I'm using `<TimeSeriesChart>` for the first time here! It powers this chart and now the scheduled/delivered trips chart, too.

## Testing Instructions

Visit the Service pages for Red, Orange, Blue and confirm that this graph loads and renders correctly.
